<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserStudyAppClass</class>
 <widget class="QMainWindow" name="UserStudyAppClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Study</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="screens">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="WelcomeScreen">
       <attribute name="title">
        <string>Welcome</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QFrame" name="welcomeFrame">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>895</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <layout class="QVBoxLayout" name="welcomeLayout">
             <property name="margin">
              <number>100</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Welcome to the &amp;quot;name&amp;quot; study!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;This study &lt;/span&gt;&lt;span style=&quot; font-family:'Arial,Helvetica,sans'; font-size:9pt; color:#000000;&quot;&gt;Fusce facilisis commodo rutrum. Maecenas id dui ac eros molestie laoreet at a urna. Suspendisse lorem urna, aliquam ac dignissim sed, molestie sit amet nibh.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Resources/userstudyResource.qrc">:/UserStudyApp/logo.svg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="1" rowspan="2">
            <widget class="QPushButton" name="nextButtonWelcome">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>40</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #b6d14c, stop:1 #83a01c);
	color: white;
	border: 1px solid #97B134;
	font-family: 'Open Sans', sans-serif;
	font-size: 16px;
    border-radius: 5px;
}

QPushButton:hover{
    background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #bbda4c, stop:1 #83a01d);
}

QPushButton:pressed{
   background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #83a01c, stop:1 #b6d14c);
}</string>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/userstudyResource.qrc">
               <normaloff>:/UserStudyApp/nextPage.ico</normaloff>:/UserStudyApp/nextPage.ico</iconset>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TutorialScreen">
       <attribute name="title">
        <string>Tutorial</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="tutorialFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>895</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <layout class="QVBoxLayout" name="tutorialLayout">
             <item>
              <widget class="Phonon::VideoPlayer" name="videoPlayer">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="nextButtonTutorial">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #b6d14c, stop:1 #83a01c);
	color: white;
	border: 1px solid #97B134;
	font-family: 'Open Sans', sans-serif;
	font-size: 16px;
    border-radius: 5px;
}

QPushButton:hover{
    background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #bbda4c, stop:1 #83a01d);
}

QPushButton:pressed{
   background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #83a01c, stop:1 #b6d14c);
}</string>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/userstudyResource.qrc">
               <normaloff>:/UserStudyApp/nextPage.ico</normaloff>:/UserStudyApp/nextPage.ico</iconset>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="DesignScreen">
       <attribute name="title">
        <string>Design</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QFrame" name="DesignFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="1">
            <widget class="QPushButton" name="nextButtonDesign">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #b6d14c, stop:1 #83a01c);
	color: white;
	border: 1px solid #97B134;
	font-family: 'Open Sans', sans-serif;
	font-size: 16px;
    border-radius: 5px;
}

QPushButton:hover{
    background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #bbda4c, stop:1 #83a01d);
}

QPushButton:pressed{
   background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #83a01c, stop:1 #b6d14c);
}</string>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/userstudyResource.qrc">
               <normaloff>:/UserStudyApp/nextPage.ico</normaloff>:/UserStudyApp/nextPage.ico</iconset>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>895</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <layout class="QVBoxLayout" name="designLayout">
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="EvaluateScreen">
       <attribute name="title">
        <string>Evaluate</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QFrame" name="evaluateFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="1">
            <widget class="QPushButton" name="nextButtonEvaluate">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #b6d14c, stop:1 #83a01c);
	color: white;
	border: 1px solid #97B134;
	font-family: 'Open Sans', sans-serif;
	font-size: 16px;
    border-radius: 5px;
}

QPushButton:hover{
    background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #bbda4c, stop:1 #83a01d);
}

QPushButton:pressed{
   background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #83a01c, stop:1 #b6d14c);
}</string>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
             <property name="icon">
              <iconset resource="Resources/userstudyResource.qrc">
               <normaloff>:/UserStudyApp/nextPage.ico</normaloff>:/UserStudyApp/nextPage.ico</iconset>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>895</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <layout class="QVBoxLayout" name="evaluateLayout">
             <item>
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="FinishScreen">
       <attribute name="title">
        <string>Finish</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>40</number>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <item row="2" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="leftMargin">
           <number>200</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>200</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="sendResultButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #b6d14c, stop:1 #83a01c);
	color: white;
	border: 1px solid #97B134;
	font-family: 'Open Sans', sans-serif;
	font-size: 24px;
    border-radius: 5px;
}

QPushButton:hover{
    background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #bbda4c, stop:1 #83a01d);
}

QPushButton:pressed{
   background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #83a01c, stop:1 #b6d14c);
}</string>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Alternatively, you can send us the results file to &lt;/span&gt;&lt;a href=&quot;mailto:space.saver@gmail.com?Subject=Results&quot;&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline; color:#0000ff;&quot;&gt;graphics.user.study@gmail.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Thank you!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Please make sure you submit your results by clicking the send button bellow:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="leftMargin">
           <number>200</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>200</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="saveResultButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-family: 'Open Sans', sans-serif;
font-size: 16px;</string>
            </property>
            <property name="text">
             <string>Save to File..</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/userstudyResource.qrc"/>
 </resources>
 <connections/>
</ui>
